gmodule-2.0.pc adds -Wl,--export-all-symbols, which not everything
using enchant needs (or wants, e.g. webkit uses dllexport).

--- origsrc/enchant-1.4.2/enchant-uninstalled.pc.in	2008-05-05 15:19:22.000000000 -0500
+++ src/enchant-1.4.2/enchant-uninstalled.pc.in	2009-06-18 13:28:11.070637800 -0500
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: libenchant
 Description: A spell checking library
 Version: @VERSION@
-Requires: glib-2.0 gmodule-2.0
+Requires: glib-2.0 gmodule-no-export-2.0
 Libs: ${pc_top_builddir}/${pcfiledir}/src/libenchant.la
 Cflags: -I${pc_top_builddir}/${pcfiledir}/src
--- origsrc/enchant-1.4.2/enchant.pc.in	2008-05-05 15:19:22.000000000 -0500
+++ src/enchant-1.4.2/enchant.pc.in	2009-06-18 13:28:16.445719900 -0500
@@ -6,6 +6,6 @@ includedir=@includedir@
 Name: libenchant
 Description: A spell checking library
 Version: @VERSION@
-Requires: glib-2.0 gmodule-2.0
+Requires: glib-2.0 gmodule-no-export-2.0
 Libs: -L${libdir} -lenchant
 Cflags: -I${includedir}/enchant
